BraveRewardsUI/Publisher/PublisherSummaryView.swift

```diff
@@ -68,6 +75,45 @@ class PublisherSummaryView: UIView {
       $0.height.equalTo(40.0)
     }
   }
+  
+  func addMonthlyTipView() {
+    if monthlyTipView == nil {
+      monthlyTipView = DetailActionableRow().then {
+        $0.textLabel.text = Strings.TipSiteMonthly
+        let batAttrString = NSAttributedString(
```

Comment:
Assuming refactored `DetailActionableRow` becomes a Button:
- Just add a `CurrencyContainerView` to the `DetailActionableRow` instead of dealing with attributed strings
- Don't see a reason to use `BATTipOption` here, if anything `BATValue` should be used but not sure if thats needed either

Reply:
Didn't realise they are there. Thanks!